source/localizable/components/wrapping-content-in-a-component.md

```diff
@@ -105,9 +113,26 @@ When `{{post.body}}` is instantiated, it will have both the edit style and the `
 ```app/templates/index.hbs
 {{#blog-post editStyle="markdown-style" as |post|}}
   <p class="author">by {{author}}</p>
-  {{post.body editStyle="compact-style"}}
+  {{post.body bodyStyle="compact-style"}}
+{{/blog-post}}
+```
+
+Alternatively, `postData` can be passed to `blog-post` at the block level if it was made available outside the component block as `myText`.
```

Comment:
I don't think we need to give both alternatives here.  Just use this as the main example (like commented above).  It distracts from what the main point of the section is.